tests/crypto/prj.conf

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# TEST
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=65536
+CONFIG_MAIN_STACK_SIZE=8096
+CONFIG_TEST_USERSPACE=n
+CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE=y
+
+# LOG
+CONFIG_LOG=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+CONFIG_LOG_IMMEDIATE=y
+CONFIG_LOG_DEFAULT_LEVEL=4
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_HAS_SEGGER_RTT=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_BLOCK=y
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=16384
+CONFIG_LOG_BACKEND_SHOW_COLOR=n
+CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=n
+
+# MISC
+CONFIG_DK_LIBRARY=y
+CONFIG_FLOAT=y
+CONFIG_MAIN_STACK_SIZE=8096
+CONFIG_SPEED_OPTIMIZATIONS=y
+CONFIG_SYS_POWER_MANAGEMENT=n
+
+# CRYPTO
+CONFIG_NORDIC_SECURITY_BACKEND=y
+CONFIG_ENTROPY_CC310=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_MBEDTLS_TLS_LIBRARY=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=65536
```

Comment:
65k is a lot, is this necessary?

Reply:
Reduced in 7aee7a9

Reply:
Updated: The original reduction was to 16 kB.
This works with hw backend, but not sw.
After some testing, sw requires 32 kB or will fail on allocs at later stages in the test.

It is not obvious to me why such a large heap is required. I see no obvious memleaks (but I don't rule it out).
Another possibility is that the heap gets fragmented somehow.

For now: 16 kB -> 32 kB in adc45bd
(The other heap/stack configs changed earlier are still reduced)